src/styles/primeng-theme/components/_common.scss

```diff
@@ -564,17 +564,23 @@ textarea,
             background-color: #f6f8fa;
             padding: 8px;
         }
-        td a {
-            border: 0 none;
-            text-align: center;
-            padding: 8px;
-            &.ui-state-highlight {
-                background-color: #d6d6d6;
-                color: #212121;
+        td {
+            a {
+                border: 0 none;
+                text-align: center;
+                padding: 8px;
+                &.ui-state-highlight {
+                    background-color: #d6d6d6;
```

Comment:
Where all this colors are comming from? no vars?

Reply:
legacy colors, prob created before the existence of color variables